collect_app/src/main/java/org/odk/collect/android/activities/GeoPointActivity.java

```diff
@@ -192,7 +192,11 @@ public void onClientStartFailure() {
 
     @Override
     public void onClientStop() {
-
+        locationClient.stopLocationUpdates();
```

Comment:
This method is called from AndroidLocationClient.stop() which also calls stopLocationUpdates(), so this line here seems to be redundant (we call it twice).

Reply:
You are correct in the common case but it looks like `onClientStop` can get called directly in some cases (`GoogleLocationClient.onConnectionSuspended` in particular). I'd rather keep this call explicitly here since it's very cheap. Does that seem ok, @grzesiek2010?

Reply:
Yes that's right.